### ISSUE TITLE
Get nan loss during training

### ISSUE THREAD

**Issue by auroua** [facebookresearch/maskrcnn-benchmark#841]
## ❓ Questions and Help
I am training a model modified from  `maskrcnn-benchmark`. When the model is trained using single `GPU` , the loss is correctly, but when trained using 4 `gpus` the model is quite easy get `nan`. How to solve this problem?

**Comment by miguelvr** [comment]
try lowering the learning rate

**Comment by leijiezhang** [comment]
just lower your learning rate or augment your training set

**Comment by leijiezhang** [comment]
or to adjust your batch size and make it larger

**Comment by yuleichin** [comment]
I met the same problem. Trained with 4 GPU, the loss is nan from the first few iterations.

**Comment by auroua** [comment]
This problem is solved by set the parameter `WARMUP_ITERS` to 1000.
